config/config.go

```diff
@@ -181,6 +189,20 @@ func FromFlags() (*Config, error) {
 	}
 	cfg.FilterExpiry = exp
 
+	if accessSporkHosts != "" {
+		heightHosts := strings.Split(accessSporkHosts, ",")
+		for _, hh := range heightHosts {
+			v := strings.Split(hh, "@")
+			heightVal, host := v[0], v[1]
```

Comment:
it's a good idea to check that `len(v) == 2` and return a helpful error message. otherwise this will panic

Reply:
yeah could add more validation